assets/company/logo-fandom-powered-by-wikia.svg

```diff
@@ -1,6 +1,6 @@
 <svg width="254" height="26" viewBox="0 0 254 26" xmlns="http://www.w3.org/2000/svg">
   <defs>
-    <linearGradient x1="0%" y1="100%" x2="63.848%" y2="32.54%" id="logo-fandom-powered-by-wikia-gradient">
+    <linearGradient x1="0%" y1="100%" x2="63.848%" y2="32.54%" id="a">
```

Comment:
This is not a good change, I think we should manually revert / fix all those ids to be unique.